portfolio/src/main/webapp/index.html

```diff
@@ -24,8 +24,25 @@ <h1 style="background-color:SlateBlue;">Anesia's Portfolio</h1>
             Feel free to add me if you like.</p>
         <p> Also check out my <a href="https://github.com/hoil200">Github</a>
             if you so desire.</p>
-        <button onclick = "getData()">Click for a medical show</button>
-        <div id = "show-container"></div>
+        
+        <div id="comments"></div>
+
+        <body>
+    
+        <form action = "/data" method = "post">
+          <p>Leave a comment here</p>
+          <textarea name = "text-input"></textarea>
+          <br/><br/>
+          
+          <input type = "submit" />
+        </form>
+        <br/>
+        <form action = "/dataStore" method "post">
```

Comment:
Your /dataStore path is associated with the DataStore only handles a GET method so there's no need to wrap the Show Comments button in a form element. loadComments() in script.js will fetch the /dataStore path and that's enough. Please remove it.